.github/pull_request_template.md

```diff
@@ -0,0 +1,86 @@
+<!--
+Notes:
+1. Please keep the PR as small as is practicable; the larger a PR gets, the harder it becomes to review and the more time it requires to get it merged.
+2. Similarly, please avoid PRs that cover more than one type; it should be a _bug fix_ *OR* a _new feature_ *OR* a _refactor_, etc.
+3. Please direct questions to the [`#formulas` channel on Slack](https://saltstackcommunity.slack.com/messages/C7LG8SV54/), which is bridged to `#saltstack-formulas` on Freenode.
+4. Feel free to suggest improvements to this template by reporting an issue or submitting a PR.  The source of this template is:
+  - https://github.com/saltstack-formulas/.github/blob/master/.github/pull_request_template.md
+-->
```

Comment:
Maybe add a note `this part can be removed`?

Reply:
@n-rodriguez Since it's a comment, it doesn't need to be removed.  It doesn't show up when creating a PR (or issue).